Problem-2.cpp

```diff
@@ -0,0 +1,93 @@
+// Time Complexity : O(N) 
```

Comment:
you are using a bucket size of 1000 which you can say as constant

Reply:
okay. got it!